src/Cms/Content.php

```diff
@@ -115,7 +115,7 @@ public function convertTo(string $to): array
 			$oldField = $oldFields->get($name);
 
 			// field name and type matches with old template
-			if ($oldField && $oldField->type() === $newField->type()) {
+			if ($oldField?->type() === $newField->type()) {
```

Comment:
TBH I think this is less clear now. Previously you could see on the first glance that this `if` only applies if there is an `$oldField`. Now you need to spot that question mark.

Reply:
The check s there to see if old and new field have the same type. I think it's rather clear that if oldField does not exist the type wouldn't be the same even if we don't first check explicitly for oldField on its own.